repository.yaml

```diff
@@ -1,2 +1,2 @@
+digital-service: API Platform
 repoVisibility: public_0C3F0CE3E6E6448FAD341E7BFA50FCD333E06A20CFF05FCACE61154DDBBADF71
-digital-service: MTD
```

Comment:
I think this should stay the same - check what we've done in our other public repos